README.md

```diff
@@ -85,52 +85,52 @@ Apollo Engineering team. Your safety is our #1 priority,
 
 Apollo 3.5 is capable of navigating through complex driving scenarios such as residential and downtown areas. The car now has 360-degree visibility, along with upgraded perception algorithms to handle the changing conditions of urban roads, making the car more secure and aware. Scenario-based planning can navigate through complex scenarios, including unprotected turns and narrow streets often found in residential areas and roads with stop signs.
 
-![image alt text](docs/demo_guide/images/Apollo_3_5_Architecture.png)
+![](docs/demo_guide/images/Apollo_3_5_Architecture.png)
 
 [**Apollo 5.0:**](docs/quickstart/apollo_3_5_quick_start.md)
 
 Apollo 5.0 is an effort to support volume production for Geo-Fenced Autonomous Driving.
 The car now has 360-degree visibility, along with upgraded perception deep learning model to handle the changing conditions of complex road scenarios, making the car more secure and aware. Scenario-based planning has been enhanced to support additional scenarios like pull over and crossing bare intersections.
 
-![image alt text](docs/demo_guide/images/Apollo_5_0_diagram1.png)
+![](docs/demo_guide/images/Apollo_5_0_diagram1.png)
 
 [**Apollo 5.5:**](docs/quickstart/apollo_5_5_quick_start.md)
 
 Apollo 5.5 enhances the complex urban road autonomous driving capabilities of previous Apollo releases, by introducing curb-to-curb driving support. With this new addition, Apollo is now a leap closer to fully autonomous urban road driving. The car has complete 360-degree visibility, along with upgraded perception deep learning model and a brand new prediction model to handle the changing conditions of complex road and junction scenarios, making the car more secure and aware. 
 
-![image alt text](docs/demo_guide/images/Apollo_5_5_Architecture.png)
+![](docs/demo_guide/images/Apollo_5_5_Architecture.png)
 
 [**Apollo 6.0:**](docs/quickstart/apollo_6_0_quick_start.md)
 
 Apollo 6.0 incorporates new deep learning models to enhance the capabilities for certain Apollo modules. This version works seamlessly with new additions of data pipeline services to better serve Apollo developers. Apollo 6.0 is also the first version to integrate certain features as a demonstration of our continuous exploration and experimentation efforts towards driverless technology.
 
-![image alt text](docs/demo_guide/images/Apollo_6_0.png)
+![](docs/demo_guide/images/Apollo_6_0.png)
 
 ## Architecture
 
 * **Hardware/ Vehicle Overview**
 
-![image alt text](docs/demo_guide/images/Hardware_overview_3_5.png)
+![](docs/demo_guide/images/Hardware_overview_3_5.png)
 
 * **Hardware Connection Overview**
 
-![image alt text](docs/demo_guide/images/Hardware_connection_3_5_1.png)
+![](docs/demo_guide/images/Hardware_connection_3_5_1.png)
 
 * **Software Overview**
 
-![image alt text](docs/demo_guide/images/Apollo_3_5_software_architecture.png)
+![](docs/demo_guide/images/Apollo_3_5_software_architecture.png)
 
 ## Installation
 
```

Comment:
Should we add Software Installation Guide here ?

Reply:
Software installation guide is referenced in quickstart. You can add software installation guide in "installation" section  to replace step 2 and 3.